AlphaWallet/Common/Services/ImageLoader.swift

```diff
@@ -17,7 +17,7 @@ protocol ImageCacheStore: AnyObject {
 }
 
 final class KingfisherImageCacheStore: ImageCacheStore {
-    private let diskStorage = ImageCache.default.diskStorage
+    private let diskStorage = ImageCache(name: "default").diskStorage
```

Comment:
Why do we need to change this?

Reply:
my bad, wrong pr, this overrides need to avoid crash as #6272 stores json as well as image in cache